sources/packages/web/src/views/student/financial-aid-application/SelectProgram.vue

```diff
@@ -109,13 +113,16 @@ export default {
       enableEducationProgram.value = true;
       selectedInstitute.value = event.value;
     };
-
+    const financialInfo = routeConstants.FINANCIALINFO;
```

Comment:
I would recommend exposing in the setup the routeConstants object instead of each route needed. Is it possible or there is some technical constraint?
If we need access to these constantly from the template, we can look into a way to declare the route constants globally .

Reply:
Good Point. Fixed